ios/Classes/NativeBridge/Connectors/PinConnector.swift

```diff
@@ -81,7 +54,7 @@ enum PinAction : String {
 
 enum PinFlow : String {
     case create = "create",
-         change = "change",
-         authentication = "authentication",
-         nextAuthenticationAttempt = "nextAuthenticationAttempt"
+//         change = "change",
```

Comment:
Commented code - if not needed please remove.

Reply:
Those enums were actually not used at all anymore :)